templates/events.html

```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+{% block content %}
+<h3>Events</h3>
+{% endblock %}
```

Comment:
The **events** page inherits from our base template and adds some custom placeholder text.